.travis.yml

```diff
@@ -11,6 +11,9 @@ env:
     - SECRET_KEY=hellowolrd@1234
     - NODE_ENV=test
     - APP_URL_FRONTEND=http://127.0.0.1:3000
+    - SOCKET_PORT=5000
+    - JAWANS_USER=ahjawans@gmail.com
+    - JAWANS_PASS=AHJ@w@nzZ
```

Comment:
These are secrets that can be abused by malicious actors, you are better off putting them in travis settings page where you can post environment secrets